sklearn/feature_selection/from_model.py

```diff
@@ -173,6 +174,10 @@ class SelectFromModel(BaseEstimator, SelectorMixin):
         Otherwise train the model using ``fit`` and then ``transform`` to do
         feature selection.
 
+    norm_order : non-zero int, inf, -inf, default 1
+        Order of the norm used to compare the vectors of coefficients in the
+        case where the coeff_ attribute of the estimator is of dimension 2.
```

Comment:
Since there is no checking of the dimension of `feature_importances_` in the code, I assumed this would always be 1 dimensional and that the `norm_order` parameter would always be irrelevant when `feature_importances_` is used.